server/routes/paths.ts

```diff
@@ -79,6 +81,9 @@ const paths = {
       home: decisionMakerAssessmentHome,
     },
   },
+  forms: {
+    pdf,
```

Comment:
when we implement forms I think this will be something like:
`/assessment/:id/forms/:form-type`

Reply:
Yes, we can do something like this once we do actual implementation.